packages/@uppy/provider-views/src/style/uppy-ProviderBrowser-viewType--list.scss

```diff
@@ -75,6 +75,7 @@
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+    user-select: none;
```

Comment:
Generally `user-select` is a bad practise. Do we really need this?

Reply:
We are debating it, yes, but without it when you click with shift all the file names get selected in blue.

Reply:
I'm pretty sure this should be possible to fix some other way. Doesn't happen for me currently on uppy.io on Brave. But if it does elsewhere, perhaps we can change the HTML somehow.

Reply:
![image](https://github.com/transloadit/uppy/assets/1199054/35123519-c06c-474c-b544-03c645205234)

Firefox, also doesn't happen in Chrome for me.